crepecake/src/integration-test/java/com/google/cloud/tools/crepecake/registry/ManifestPullerIntegrationTest.java

```diff
@@ -31,19 +30,14 @@
 
   @ClassRule public static LocalRegistry localRegistry = new LocalRegistry(5000);
 
-  @BeforeClass
-  public static void setUpLocalRegistry() throws IOException, InterruptedException {
-    localRegistry.pullBusybox();
-  }
-
   @Test
   public void testPull_v21() throws IOException, RegistryException {
     RegistryClient registryClient = new RegistryClient(null, "localhost:5000", "busybox");
-    ManifestTemplate manifestTemplate = registryClient.pullManifest("latest");
+    V21ManifestTemplate manifestTemplate =
+        registryClient.pullManifest("latest", V21ManifestTemplate.class);
 
     Assert.assertEquals(1, manifestTemplate.getSchemaVersion());
-    V21ManifestTemplate v21ManifestTemplate = (V21ManifestTemplate) manifestTemplate;
-    Assert.assertTrue(0 < v21ManifestTemplate.getFsLayers().size());
+    Assert.assertTrue(0 < manifestTemplate.getFsLayers().size());
```

Comment:
not possible to test the exact number?
If so, it's a little more conventional to swap these arguments of <

Reply:
It might not be safe to assume a constant number for the number of layers in gcr.io/distroless/java.